src/core/scene/a-scene.js

```diff
@@ -72,13 +79,11 @@ var AScene = module.exports = registerElement('a-scene', {
 
     attachedCallback: {
       value: function () {
-        initFullscreen(this);
         initMetaTags(this);
         initWakelock(this);
 
         window.addEventListener('load', this.resize.bind(this));
         window.addEventListener('resize', this.resize.bind(this), false);
-        this.addEventListener('fullscreen-exit', this.exitVR.bind(this));
```

Comment:
also, I hadn't gotten far with it - but I think we need to have an Exit VR button